src/PageContent/Calendar/DelivererGroupContent.js

```diff
@@ -0,0 +1,164 @@
+                                                defaultValue={this.state.email1}
+                                                type="email"
+                                                required
+                                            />
+                                            <input
+                                                className="content-details deliverer-group-details"
+                                                defaultValue={
+                                                    this.state.deliverer2
+                                                }
+                                                type="text"
+                                                name="deliverer2"
+                                                required
+                                            />
+                                            <input
+                                                className="content-details deliverer-group-details"
+                                                defaultValue={this.state.phone2}
+                                                type="tel"
+                                                name="phone2"
+                                                pattern={StringFormat.PHONE}
+                                                required
+                                            />
+                                            <input
+                                                className="content-details deliverer-group-details"
+                                                defaultValue={this.state.email2}
+                                                type="email"
+                                                required
+                                            />
+                                        </div>
+
+                                        <input
+                                            type="submit"
+                                            className="edit-button"
+                                            value="save"
+                                        />
+                                    </form>
+                                </div>
+                            )}
+                            {this.props.accountType ===
+                                AccountType.DELIVERER_GROUP &&
+                            !this.state.edit &&
+                            this.props.futureEvent ? (
+                                    <button
+                                        type="button"
+                                        className="edit-button"
+                                        onClick={this.edit}
+                                    >
+                                    Edit
+                                    </button>
+                                ) : null}
+                        </div>
+                    ) : (
+                        <div>To Be Determined.</div>
```

Comment:
Since we're always displaying the group's name, maybe change this to something that conveys that it's the deliverers that's tbd? ummm "Volunteers not yet assigned"? "Volunteers not assigned yet"? "Volunteers to be determined"??

Also I think we should display the text in red or something if account is DG so that it'd prompt them to do something about it more.